requirements/pip.txt

```diff
@@ -77,8 +77,6 @@ nilsimsa==0.2.0
 git+https://github.com/alex/django-filter.git#egg=django-filter
 git+https://github.com/ericflo/django-pagination.git@e5f669036c#egg=django_pagination-dev
 git+https://github.com/alex/django-taggit.git#egg=django_taggit-dev
-
 git+https://github.com/ericholscher/readthedocs-sphinx-ext#egg=readthedocs_ext
-
```

Comment:
Normally I'd say avoid whitespace changes, but that call is up to the maintainers.